_tutorials/convert-html-site-to-jekyll.md

```diff
@@ -0,0 +1,362 @@
+---
+layout: tutorials
+permalink: /tutorials/convert-html-site-to-jekyll/
+title: Convert an HTML site to Jekyll
+---
+
+If you're looking for themes for your Jekyll site, you don't have to restrict yourself to existing Jekyll themes. You can convert almost any static HTML site into a Jekyll theme. Understanding how to convert any HTML site into a Jekyll website will open your world to many more options for Jekyll themes.
+
+Although websites can have sophisticated features and controls, we'll keep things simple in this tutorial. This tutorial contains the following sections:
+
+* TOC
+{:toc}
+
+## 1. Understand a basic Jekyll site
+
+First let's start with a grounding in the basics. Stripping a Jekyll site down to an extremely basic level will help clarify what happens in a Jekyll site. If you haven't already installed the jekyll gem, [install it]({% link _docs/installation.md %}).
+
+A simple Jekyll site can consist of just 3 files:
+
+```
+├── _config.yaml
+├── _layouts
+│   └── default.html
+└── index.md
+```
+
+Create these 3 files in a folder called `myjekyllsite`. (Put `default.html` inside a folder called `_layouts`.) Then populate the content of the files as follows:
+
+**_config.yml**
+
+```yaml
+markdown: kramdown
```

Comment:
In this tutorial, I'm not having users run `jekyll new site` because that ends up installing a default gem theme with its own styles and such. I have them just create the config file manually, so nothing would be populated by default in it.